lib/SlackBot.js

```diff
@@ -344,15 +345,35 @@ function Slackbot(configuration) {
                 } else {
                     message.type = 'user_channel_join';
                 }
+                next();
             } else if (message.subtype && message.subtype == 'group_join') {
                 // someone joined. maybe do something?
                 if (message.user == bot.identity.id) {
                     message.type = 'bot_group_join';
                 } else {
                     message.type = 'user_group_join';
                 }
-            } else if (message.subtype) {
+                next();
+            }  else if (message.subtype == 'message_replied') {
+                message.type = 'message_replied';
+                message.text = message.message.text;
+                message.user = message.message.user;
+                var options = { method: 'POST',
```

Comment:
this should use bot.api.conversations.replies instead of making a direct call to request.

other than that, I think this looks good to go.

Reply:
@Oliverr Want to take a last pass at this so we can merge this in?